packages/svelte/src/reactivity/set.js

```diff
@@ -86,14 +88,28 @@ export class ReactiveSet extends Set {
 		return super.has(value);
 	}
 
+	#read_all() {
+		var sources = this.#sources;
+		var values = super.values();
+		for (let value of values) {
```

Comment:
Like in Map, you can check if the sizes of the two match before doing this.

Reply:
Gotcha...fixed it